src/layers/core/geojson-layer/geojson-layer.js

```diff
@@ -117,33 +113,29 @@ export default class GeoJsonLayer extends CompositeLayer {
     // Polygon outline or wireframe
     let polygonOutlineLayer = null;
     if (drawPolygons && extruded && wireframe) {
-      polygonOutlineLayer = new PolygonLayer(Object.assign({}, this.props, {
+      polygonOutlineLayer = new PrimitivePolygonLayer(Object.assign({}, this.props, {
         id: `${id}-polygon-wireframe`,
         data: polygonFeatures,
         getPolygon: getCoordinates,
         getElevation,
-        getColor: getStrokeColor,
+        getColor,
         extruded: true,
         wireframe: true,
         onHover,
         onClick,
-        updateTriggers: {
-          getColor: this.props.updateTriggers.getStrokeColor
-        }
+        updateTriggers: Object.assign({}, this.props.updateTriggers, {
```

Comment:
This will throw an error when an updateTrigger cannot be matched with an accessor.